src/main/java/com/kainos/ea/dao/RolesDao.java

```diff
@@ -114,4 +114,30 @@ public JobSpecification getAllSpecification(Connection c, int role_id) throws SQ
             throw new SQLException ("Select query for job specifications is incorrect");
         }
     }
-}
+
+    public int createJobRole(JobRoleRequest jobRoleXL, Connection c) throws SQLException{
+
+        int newId = 0;
```

Comment:
Hm, this name took me a second to understand - maybe something like "createdRoleId" would be clearer?